### PR TITLE
UI API Layer documentation updates

### PR DESCRIPTION
**Pull request by tomekpapiernik**
**Description**

Changes proposed in this pull request:

Updates to UI API Layer (minor).

**Related issue(s)**
#2415

### CODE REVIEW

File: docs/console/docs/03-02-ui-api-layer.md [hunk 1]
```diff
@@ -9,27 +9,27 @@ The UI API Layer is a backend service which provides an API for all views of the
...
-To enable a given module, install a proper Kyma component. It includes the BackendModule custom resource definition with the same name as the name of a given module.
+- `apicontroller` is related to the API Controller
+- `authentication` is related IDP Presets
+- `application` is related the Application Connector
+- `content` is related to documentation
+- `kubeless` is related to Serverless
+- `servicecatalog` is related to the Service Catalog, including Service Classes, Service Instances, and Service Bindings
+- `servicecatalogaddons` is related to the Service Catalog add-ons, such as ServiceBindingUsage, and UsageKinds
+
+To enable a given module, install the corresponding Kyma component. It includes the BackendModule custom resource with the same name as the name of a given module.
```
Comment (tomekpapiernik) [inline docs/console/docs/03-02-ui-api-layer.md]: Can you please confirm if this sentence is correct? I went from "BackendModule custom resource definition" to "BackendModule custom resource" as then it says that the resource has a name that's the same as the module name. I think that the CRD names are constant, but CRs created from CRDs have different names.

File: docs/console/docs/03-02-ui-api-layer.md [hunk 2]
```diff
@@ -9,27 +9,27 @@ The UI API Layer is a backend service which provides an API for all views of the
...
 
 ## Modularization
 
 The UI API Layer consists of the Kubernetes resource logic and cache for different domains, such as the Service Catalog, Application, or Kubeless. The UI API Layer introduces modularization changes which are based on toggling modules while the server is running. The enabled module synchronizes cache for its resource and enables the module's logic for all server requests. If you disable a given module, every GraphQL query, mutation, and subscription related to this module returns an error.
 
 The UI API Layer module pluggability is hidden behind a feature toggle. It is not enabled by default because the Console UI still requires resiliency improvements to ensure no errors occur when a certain Kyma component is not installed.
```
Comment (tomekpapiernik) [inline docs/console/docs/03-02-ui-api-layer.md]: I wanted to rephrase this sentence as well, but I'm not sure how to do it. My draft was: 
`The UI API Layer module pluggability must be enabled through feature toggle. You may experience errors if the component for which you want to use the GraphQL module is not installed.`
Can you verify if this is okay?

File: docs/console/docs/03-02-ui-api-layer.md (hunk 1 above)
Comment (pkosiec) [reply docs/console/docs/03-02-ui-api-layer.md]: Exactly. `BackendModule custom resource` is the correct form in this context.

File: docs/console/docs/03-02-ui-api-layer.md (hunk 2 above)
Comment (pkosiec) [reply docs/console/docs/03-02-ui-api-layer.md]: Actually, the whole paragraph can be deleted as there is no feature toggle anymore.

Comment (tomekpapiernik) [reply docs/console/docs/03-02-ui-api-layer.md]: cool, thanks :)

**Comment by tomekpapiernik** [comment]
> Anyway, apart from two fixes (CRD->CR, removing paragraph), do we really need to change such details? 😄

Two words: continuous improvement ;)

**Pull request merged**